apps/site/src/middleware.page/return-types-as-json.ts

```diff
@@ -67,7 +67,7 @@ export const returnTypeAsJson = async (request: NextRequest) => {
 
   const kind = url.match(versionedTypeUrlRegExp)?.[1];
 
-  let type: DataType | PropertyType | EntityType =
+  let type: DataType | PropertyType | EntityType | null =
```

Comment:
Wouldn't this be `undefined` rather than `null`?

Reply:
It might possibly be `undefined` here yes, but it will be turned into `null` if it doesn't become a type below. Could be improved but not today